### ISSUE TITLE
Dropdown selection box shows Keyboard 01, 02, 03.  Input monitor shows Keyboard 03,04.  Logitech keyboards

### ISSUE THREAD

**Issue by 1Coro1** [djlastnight/KeyboardSplitterXbox#102]
Great work djlastnight on a handy program that I didn't realise would be so hard to find!

When I run the Xbox 360 input test, there appears to be no input (I can't see any indication on the controller image that shows a button has been pressed).

I have 2 x Logitech K270 wireless keyboards.  I have tried with them both paired with the same USB dongle, and have also tried to different Logitech USB dongles, with 1 keyboard paired seperately with each.

I have noticed that the keyboard numbers don't match the numbers shown in the input monitor at bottom of screen....
The dropdown box where you choose what keyboard to use shows Keyboard 01, Keyboard 02, Keyboard 03.
If I use the autodetect option, sometimes it has done nothing, other times both keyboards autodetect as Keyboard 02.
However, if I check the input monitor down the bottom, then one keyboard will show as Keyboard 04, and the other will show as Keyboard 05.

So it seems the program is seeing both keyboards as different devices in the input monitor, but won't auto detect them the same.  Let me know if this dosn't make sense and I can get a screenshot.
Cheers :-)

**Comment by djlastnight** [comment]
It makes sense. Does it work well, if you get the right keyboard names/numbers from the input monitor and manually assign them to different slots?

**Comment by 1Coro1** [comment]
No, that was the first thing I tried - I have tried all combinations and can't get it to work.

I didn't explain this clearly I think...

The numbers in the drop down box never match what is being shown in the input monitor.
The dropdown box shows only shows the options 01, 02, 03, and  the input monitor only shows 04, 05, 06 no matter what keyboard/Logitech dongle combo I have tried.

I am guessing it shouldn't be the Logitech setup causing the issue, as the 2 different keyboards do appear to be recognised as seperate devices in the input monitor (04 & 05).

**Comment by djlastnight** [comment]
Hm, really strange problem. I need to see this on your pc. Let me know when we can do a team viewer session.
